client/src/main/proto/app/cash/backfila/service.proto

```diff
@@ -104,3 +104,10 @@ message CheckBackfillStatusResponse {
     COMPLETE = 3;
   }
 }
+
+service BackfilaService {
```

Comment:
Yeah, I debated whether or not we should add this from a symmetry perspective. Don't feel strongly either way, but it would be nice to support gRPC bidirectionally in the future (since we standardized on it internally).